packages/devtools_app/lib/src/memory/memory_controller.dart

```diff
@@ -697,6 +792,58 @@ class MemoryController extends DisposableController
     });
   }
 
+  AnalysesReference findAnalysesNode() {
+    for (final child in topNode.children) {
+      if (child is AnalysesReference) {
+        return child;
+      }
+    }
+    return null;
+  }
+
+  void createSnapshotEntries(Reference parent) {
+    for (final snapshot in snapshots) {
+      final snapShotMatch = parent.children.where((element) {
```

Comment:
nit: r/snapShotMatch/snapshotMatch to keep case consistent throughout

Reply:
Done.